ansible/group_vars/hypervisors

```diff
@@ -0,0 +1,38 @@
+---
+# Map physical network names to their source device. This can be either an
+# existing interface or an existing bridge.
+physnet_mappings: {}
+
+virtualisation_provider: libvirt
```

Comment:
Not seeing this used anywhere. Could just use the group membership for this?

Reply:
Yep, another artifact from something else - thanks.